crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -153,12 +186,11 @@ impl BundleFactory {
     /// Immediately the currently aggregating bundle.
     ///
     /// Returns an empty bundle if there are no bundled transactions.
-    pub(super) fn pop_now(&mut self) -> Vec<Action> {
+    pub(super) fn pop_now(&mut self) -> SizedBundle {
         self.finished
             .pop_front()
             .or_else(|| Some(self.curr_bundle.flush()))
-            .map(SizedBundle::into_actions)
-            .unwrap_or_default()
+            .unwrap_or(SizedBundle::new(self.curr_bundle.max_size))
```

Comment:
why is this changed? it looks like `into_actions()` is always called on the returned value?

Reply:
Popping returns a `SizedBundle` now, which contains info that we use for tracing (instruments `Executor.submit_bundle()`).
`into_actions` is called in all the tests since we dont care about the byte count and rollup counts there.